framework/helpers/BaseJson.php

```diff
@@ -109,6 +109,31 @@ public static function decode($json, $asArray = true)
         return $decode;
     }
 
+    /**
+     * Validates the given data and returns a boolean indicating whether
+     * it is a valid JSON string or not.
+     *
+     * @param mixed $data data to validate.
+     * @return boolean `true` if given data is a valid JSON string, `false` otherwise.
+     *
+     * @since 2.0.14
+     */
+    public static function validate($data)
+    {
+        $isValid = false;
+
+        if (is_string($data) === true) {
+            json_decode($data);
```

Comment:
I'd add the STFU operator just for the case.

Reply:
Nope. It doesn't raise any errors.